Comply with DITA - Mng content - Lifecycles (#4551)
theforeman/foreman-documentation: guides/common/modules/proc_adding-lifecycle-environments-by-using-web-ui.adoc

:_mod-docs-content-type: PROCEDURE

[id="adding-lifecycle-environments-by-using-web-ui"]
= Adding lifecycle environments to {SmartProxyServers} by using {ProjectWebUI}

[role="_abstract"]
If your {SmartProxyServer} has the content functionality enabled, you must add an environment so that {SmartProxy} can synchronize content from {ProjectServer} and provide content to host systems.

Do not assign the _Library_ lifecycle environment to your {SmartProxyServer} because it triggers an automated {SmartProxy} sync every time the CDN updates a repository.
This might consume multiple system resources on {SmartProxies}, network bandwidth between {Project} and {SmartProxies}, and available disk space on {SmartProxies}.

You can add an environment from the {ProjectWebUI}.

.Procedure

> . In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*, and select the {SmartProxy} that you want to add a lifecycle to.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
. Select your {SmartProxyServer} that you want to add a lifecycle to.
```

. Click *Edit* and click the *Lifecycle Environments* tab.
. From the left menu, select the lifecycle environments that you want to add to {SmartProxy} and click *Submit*.

> . To synchronize the content on the {SmartProxy}, click the *Overview* tab and click *Synchronize*.

Reviewer: ```suggestion
. To synchronize the content to your {SmartProxyServer}, click the *Overview* tab and click *Synchronize*.
```
